macro.h

```diff
@@ -547,7 +548,8 @@ int out_len = 0;\
         _07_03_ENCODE_SIGNING_PUBKEY_NULL  (__buf_out__                                 );      /* pk      | size  35 */ \
         _08_01_ENCODE_ACCOUNT_SRC          (__buf_out__, __acc__                        );      /* account | size  22 */ \
         _08_03_ENCODE_ACCOUNT_DST          (__buf_out__, to_address                     );      /* account | size  22 */ \
-        etxn_details((uint32_t)__buf_out__, PREPARE_PAYMENT_SIMPLE_TRUSTLINE_SIZE);             /* emitdet | size 1?? */  \
+        int64_t __buf_size__ = PREPARE_PAYMENT_SIMPLE_TRUSTLINE_SIZE - (__buf_out__ - buf_out_master);                   \
```

Comment:
err, did you test this? I'd expect this change to break the macro (as described in https://github.com/XRPL-Labs/xrpld-hooks/issues/64#issuecomment-1359105907 )

Reply:
No, tested the other macro only... Will check it.